fairscale/nn/data_parallel/fsdp_optim_utils.py

```diff
@@ -165,7 +201,17 @@ def build_unflat_state_dict(
     uncollected_opt_state: Dict[int, Dict],
     param_groups: List[Dict],
 ) -> Dict:
-    """Build an unflattened optimizer state dict given a list of flattened optimizer state dicts from each rank."""
+    """Build an unflattened optimizer state dict given a list of flattened optimizer state dicts
+    from each rank. This is only called on rank 0.
+
+    Args:
+        instance_list: list of FSDP wrapper objects
+        world_pad_info: [rank][fsdp_instance_id][bytes_padded_per_param]
```

Comment:
Perhaps copy the description change form above?

Reply:
updated.